web/homepage.html

```diff
@@ -1,41 +1,29 @@
 <!doctype html>
 <html lang="en">
+
 <head>
-  <meta charset="utf-8">
-  <title>Open in LBRY</title>
-  <style>
-    body {
-      max-width: 800px;
-      margin: 30px auto 0;
-      font-size: 24px;
-      line-height: 1.5;
-    }
-    #name {
-      font-size: 24px;
-      margin-bottom: 20px;
-    }
-  </style>
+
+  <meta charset="utf-8" />  
+  <title>Open in LBRY &sol; Abrir em LBRY</title>  
```

Comment:
Yes I would suggest we don't mix 2 languages on the same page. Could you change this one back to English only?